source/sync/details/conflict-resolution.txt

```diff
@@ -95,10 +100,25 @@ Strings
 ~~~~~~~
 
 Device Sync interprets the value of a string as a whole and does not merge
-conflicts on a per-character basis. For example, this means that if a character 
+conflicts on a per-character basis. For example, this means that if a character
 or substring is inserted or deleted within a string, Device Sync will treat
 this as a replacement of the entire value of the string.
 
+Dictionaries
+~~~~~~~~~~~~
+
+Generally, dictionary conflicts are handled similarly to lists. However, there
```

Comment:
We're referring to how conflicts are handled in lists, but we don't have a "Lists" section that spells this out. Instead, the information is scattered throughout the page. I might skip the intro sentence here and go straight to:

> Device Sync considers the removal of a dictionary key to be an update, rather than a deletion. As a result...

Alternately, we could add a List section that spells out list details.